legacyCode/index.html

```diff
@@ -4,35 +4,140 @@
-    <form id="inputForm" class="form-inline start-input">
-      <div class="form-group">
+  <form id="inputForm" class="start-input">
+    <div id="inputRow" class="row row-position">
+      <span id="loading" class="loading-position"></span>
+            <span class="input-group col-md-8 col-md-offset-2">
         <input type="text" class="form-control" id="searchTerm" placeholder="Type name of repository" autofocus required
                minlength="3">
-        <button id="buttonSearch" type="submit" class="btn btn-default">Search</button>
-      </div>
-    </form>
+        <span id="spanButtonSearch" class="input-group-btn"><button id="buttonSearch" type="submit"
+                                                                    class="btn btn-default">Search</button></span>
+      </span>
+      <span class="col-md-offset-2" id="message"></span>
+    </div>
+  </form>
+  <div>
+    <ul id="list" class="col-md-12">
+    </ul>
   </div>
 </div>
-<div id="loading"></div>
-<div>
-  <ul id="list" class="col-md-6">
-  </ul>
+
+<!-- Modal -->
+<div class="modal fade" id="myModal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel">
+  <div class="modal-dialog" role="document">
+    <div class="modal-content">
+      <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span
+          aria-hidden="true">&times;</span></button>
+        <h4 class="modal-title" id="myModalLabel">Information about repository</h4>
+      </div>
+
+      <div class="modal-body" id="modalBody">
+        Loading...
+      </div>
+
+      <div class="modal-footer">
+        <button type="button" class="btn btn-default" data-dismiss="modal">Close</button>
+      </div>
+    </div>
+  </div>
 </div>
 
+<script type="text/javascript"
+        src="https://cdnjs.cloudflare.com/ajax/libs/handlebars.js/4.0.5/handlebars.min.js"></script>
+<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.1.1/jquery.js"></script>
+<script type="text/javascript"
+        src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.0.0-alpha/js/bootstrap.js"></script>
 
 <script src="./app.js"></script>
 <script src="./httpService.js"></script>
 <script src="./repoModel.js"></script>
+<script src="./repoDetails.js"></script>
+
+<!--block-template-->
+<script id="block-template" type="text/x-handlebars-template">
+  <div>
+    <div class="thumbnail">
+      <div class="caption">
+        <div class="row">
+          <div class="col-md-2 visible-md"><img class="image-position" src="{{Repo.owner.avatar_url}}" alt="..."></div>
+          <div class="col-md-2 col-md-offset-1"><h4>Info</h4></div>
+          <label class="col-md-2 col-md-offset-5">{{repoDate}}</label>
+        </div>
+        <form class="form-horizontal">
+          <div class="form-group">
+            <div class="col-sm-2 col-md-offset-3">Owner's login:</div>
+            <div class="col-sm-7">
+              {{Repo.owner.login}}
```

Comment:
Use model methods in templates, i.e. `repo.getName()` etc...
Please fix for all templates.